pkg/store/proxy_test.go

```diff
@@ -573,6 +573,45 @@ func TestProxyStore_LabelValues(t *testing.T) {
 	testutil.Equals(t, 1, len(resp.Warnings))
 }
 
+func TestProxyStore_LabelNames(t *testing.T) {
```

Comment:
could you add tests with partial repsonse enabled ?

Reply:
Agree!